src/ArrayHelper.php

```diff
@@ -344,6 +344,11 @@ public static function setValue(array &$array, $path, $value): void
             return;
         }
 
+        if (is_array($array) && is_string($path) && (isset($array[$path]) || array_key_exists($path, $array))) {
```

Comment:
```suggestion
        if (is_array($array) && is_string($path) && isset($array[$path])) {
```
`isset` and `array_key_exists` are doing the same stuff in this case. That is the reason `array_key_exists` is deprecated since php 7.4 and will be removed in the future versions.

Reply:
https://www.php.net/manual/en/migration74.deprecated.php#migration74.deprecated.core.array-key-exists-objects
https://www.php.net/manual/en/function.array-key-exists.php#example-6264

Reply:
`array_key_exists()` is not deprecated, the deprecation is using the function with objects.

https://www.php.net/manual/en/migration74.deprecated.php#migration74.deprecated.core.array-key-exists-objects

Reply:
My fault, thnx.